backend/src/app/api.py

```diff
@@ -74,6 +74,6 @@ class Meta:
     load_instance = True
```

Comment:
make the following changes to PropertySchema
class PropertySchema(SQLAlchemyAutoSchema):